go/vt/vtgate/executor_select_test.go

```diff
@@ -3680,7 +3682,7 @@ func TestSelectAggregationNoData(t *testing.T) {
 		{
 			sql:         `select col, count(*) from user group by col limit 2`,
 			sandboxRes:  sqltypes.MakeTestResult(sqltypes.MakeTestFields("col|count(*)", "int64|int64")),
-			expSandboxQ: "select col, count(*), weight_string(col) from `user` group by col, weight_string(col) order by col asc limit :__upper_limit",
+			expSandboxQ: "select col, count(*), weight_string(col) from `user` group by col, weight_string(col) order by col asc",
```

Comment:
I believe this is the right thing to do, it would mean more data but accurate results.